JustSaying.IntegrationTests/Fluent/MessagingBusBuilderTests.cs

```diff
@@ -0,0 +1,271 @@
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using JustSaying.Fluent;
+using JustSaying.Messaging;
+using JustSaying.Messaging.MessageHandling;
+using JustSaying.Messaging.Monitoring;
+using JustSaying.Models;
+using JustSaying.TestingFramework;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Shouldly;
+using Xunit.Abstractions;
+
+namespace JustSaying.IntegrationTests
+{
+    public class MessagingBusBuilderTests
+    {
+        public MessagingBusBuilderTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        private ITestOutputHelper OutputHelper { get; }
+
+        [AwsFact]
+        public async Task Can_Create_Messaging_Bus_Fluently_For_A_Queue()
+        {
+            // Arrange
+            var services = new ServiceCollection()
+                .AddLogging((p) => p.AddXUnit(OutputHelper))
+                .AddJustSaying(
+                    (builder) =>
+                    {
+                        builder.Client((options) => options.WithBasicCredentials("accessKey", "secretKey").WithServiceUri(TestEnvironment.SimulatorUrl))
+                               .Messaging((options) => options.WithRegions("eu-west-1"))
```

Comment:
Why `(options) =>` instead of `options =>` or `o =>`

Reply:
Purely because that's how I like to write code, and it also makes the code's intention clearer at this exact point while it serves as the only documentation.

Reply:
YMMV 🤷‍♂️